resources/sql/queries.sql

```diff
@@ -485,9 +485,10 @@ FROM application_event
 ORDER BY id DESC
 LIMIT 1;
 
--- :name add-application-event! :insert
+-- :name add-application-event! :? :1
```

Comment:
How about using `:returning-execute` here. This doesn't seem to be correct.

https://www.hugsql.org/#retrieving-last-inserted-id-or-record

Reply:
I had some weird sql problems and was iterating stuff, this got left over from that I guess